### ISSUE TITLE
Pass extra parameters to FileSystem

### ISSUE THREAD

**Issue by oskretc** [oskretc/fman-flatview#2]
Hi @mherrmann 

I wonder if you have a mechanism to pass extra parameters to a FileSystem

e.g.

My Flat(FileSystem) expect an URL in the form of **flat://c:/tmp**

let's say I want to add some filtering. 
The only way so far I can think of is by using an URL like: **flat://c:/tmp?*.txt** 
where everything after ? will be the filter
This might not be the most efficient way.

Any other options?

**Comment by mherrmann** [comment]
Hi @oskretc! The way you suggested is what I would use. This reminds me of [fman/#41](https://github.com/fman-users/fman/issues/41) in case you haven't seen it yet.

**Comment by oskretc** [comment]
Yes, fman/#41 will definitely be much better, just want it to have something similar in the meantime. I'm also aware of Fuzzy plugin, but don't think working in the quick search window is the best.

I have tried using the above example, but I keep hitting problems when the FileSystem Class is called from multiple places. I don't have the full picture ;)

Anyway, the current release is a bit more usable, once you see the file you are interested in, hitting enter will take you to the folder and highlight the file so you can operate on it.

Have a nice weekend.

**Comment by mherrmann** [comment]
> once you see the file you are interested in, hitting enter will take you to the folder and highlight the file so you can operate on it.

That sounds good!

You too thanks :-)
